### DISCUSSION TITLE
How can I get my postgrest version

### DISCUSSION THREAD

**Discussion by neves** [PostgREST/postgrest#2551] [Q&A] [answered]

**Comment by wolfgangwalther** [comment, marked as answer]
You can run `postgrest --help` - it will show the version you have. For an already running instance, you can also have a look at the OpenApi output at `GET /` (the root endpoint) of your api.

**Reply by steve-chavez** [reply]
Also any response has a `Server` header with the version. This header might be hidden by your proxy though.
